src/Cdn/docs/Start-AzFrontDoorCdnProfileMigrate.md

```diff
@@ -0,0 +1,170 @@
+---
+external help file:
+Module Name: Az.Cdn
+online version: https://docs.microsoft.com/powershell/module/az.cdn/start-azfrontdoorcdnprofilemigrate
+schema: 2.0.0
+---
+
+# Start-AzFrontDoorCdnProfileMigrate
+
+## SYNOPSIS
+Commit the migrated Azure Frontdoor(Standard/Premium) profile.
+
+## SYNTAX
+
+```
+Start-AzFrontDoorCdnProfileMigrate -ProfileName <String> -ResourceGroupName <String>
```

Comment:
I'd like suggest naming commands from *-AzFrontDoorCdnProfileMigrate  to *-AzFrontDoorCdnProfileMigration to follow the Verb-Noun format

Reply:
Thanks, I would fix this.